draft-ietf-lamps-x509-policy-graph.md

```diff
@@ -749,9 +759,37 @@ This document has no IANA actions.
 
 --- back
 
+# Implementation Status
```

Comment:
Should this be earlier? https://datatracker.ietf.org/doc/html/rfc7942#section-2 says to put it just before Security Considerations. I assume we also should add a sentence asking the RFC editor to remove it later.

Reply:
sure, done